### ISSUE TITLE
Support for FRAM chips ?

### ISSUE THREAD

**Issue by rajibgupta** [Marzogh/SPIMemory#146]
![GitHub issue state](https://img.shields.io/github/issues/detail/s/Marzogh/SPIFlash/146.svg) ![GitHub issue title](https://img.shields.io/github/issues/detail/title/Marzogh/SPIFlash/146.svg)
![GitHub issue author](https://img.shields.io/github/issues/detail/u/Marzogh/SPIFlash/146.svg) ![GitHub issue label](https://img.shields.io/github/issues/detail/label/Marzogh/SPIFlash/146.svg) ![GitHub issue comments](https://img.shields.io/github/issues/detail/comments/Marzogh/SPIFlash/146.svg)
![GitHub issue age](https://img.shields.io/github/issues/detail/age/Marzogh/SPIFlash/146.svg) ![GitHub issue last update](https://img.shields.io/github/issues/detail/last-update/Marzogh/SPIFlash/146.svg)
###### ---------------------------- **DO NOT DELETE OR EDIT** anything above this line ----------------------------

Since FRAM chips are more stable and comes with more read/write cycles, I would like to prefer that.

Does this library support any FRAM chip ?

**Comment by Marzogh** [comment]
Hi mate, I'm working on FRAM support into the next major release. In the meantime, feel free to keep an eye on the code in the development branch. As of today it supports the FM25W256 from Cypress. The documentation for this is not ready yet so you'll have to dig through the library to figure out how it works.

I've tried to keep it simple and identical to the API that's used to access SPI Flash memory, so you won't need to make big changes to your pre-existing code - but be warned, it is a work in progress.

Do not depend on code in the _development_ branch for production ready stuff as new commits might break existing functionality. Once I've got the API properly baked in, I'll roll the code into the _stable_ branch while I work on documentation. 😄
